src/number_format.js

```diff
@@ -181,7 +182,7 @@ class NumberFormat extends React.Component {
     }
 
     //add negation back
-    if(hasNegation) num = '-' + num;
+    if (hasNegation) num = '-' + num;
```

Comment:
Float string should have always minus sign instead of brackets. Bracket should be only formatting detail.

Reply:
I agree.
I've confirmed this is the case. I have the parentheses removal logic in removePrefixAndSuffix so the brackets should never get to the underlying functions, they'll be removed/added in the removeFormatting and formatNumString functions